examples/00-postprocessing/README.txt

```diff
@@ -1,4 +1,4 @@
 Postprocessing examples
-========================
+=======================
```

Comment:
This heading issue, plus removing the redundant checked in script file in `source/examples/00-postprocessing` fixed the final doc build problems.